fuzz-tests/geometry/PointInsideHull.cpp

```diff
@@ -11,22 +11,31 @@ typedef pair<int, int> pii;
 typedef vector<int> vi;
 
 #include "../../content/geometry/ConvexHull.h"
+namespace ignore {
+	#include "../../content/geometry/onSegment.h"
+	#include "../../content/geometry/SegmentDistance.h"
+}
+template<class P> bool onSegment(P s, P e, P p) {
+	return p.cross(s, e) == 0 && (s - p).dot(e - p) <= 0;
+}
 #include "../../content/geometry/PointInsideHull.h"
 #include "../../content/geometry/insidePolygon.h"
+ostream &operator<<(ostream &os, P p) { return cout << "(" << p.x << "," << p.y << ")"; }
 
 int main() {
 	rep(it,0,100000) {
 		int N = rand() % 15;
 		vector<P> ps, ps2;
 		rep(i,0,N) ps.emplace_back(rand() % 20 - 10, rand() % 20 - 10);
 		trav(i, convexHull(ps)) ps2.push_back(ps[i]);
-		if (ps2.empty()) continue;
+		if (sz(ps2) <= 0) continue;
```

Comment:
FWIW I'd say that `ps2.empty()` is clearer

Reply:
Leftover from debugging some bugs with the new impl.